can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
I removed the `Observation.ignore` and the test for `.value functions should not be observable` still passed, so I assumed it was being taken care of by the `if` statement in `getCurrent`, but even if I remove that `if` statement it still passes, so I'm less confident also.